Expand on the privacy risks of proxy handling. (#60)
WICG/private-network-access: explainer.md

served on a public IP address, is accessed by a browser via a proxy on a private
IP address (e.g. `192.168.1.123`), then the resource will be considered to have
been fetched from a private IP address. The resource will in turn be allowed to
make requests to other private IP addresses accessible to the browser. This can
allow `foo.example` to learn that it was proxied by observing that it is allowed
to make requests to [=private addresses=], which is a privacy information leak.

> Note however that this requires correctly guessing the URL of a resource on the
> private network. On the other hand a single correct guess is enough.

Reviewer: ```suggestion
While this requires correctly guessing the URL of a resource on the private
network, a single correct guess is sufficient.
```

I was going to suggest adding some commas, but realized perhaps this can just be reworded. I'm also fine with the original wording, just with more commas as appropriate ;)

Author: Thanks for the suggestion, it's a great improvement.
